src/main.rs

```diff
@@ -355,9 +355,13 @@ impl Application {
                     // Do nothing for now.
                 },
                 Event::FocusGained => {
+                    let mut store = self.store.lock().await;
+                    store.application.focused = true;
```

Comment:
Do we always get this event on startup?

Reply:
Sorry busy week.
Thanks for the catch @benjajaja, I tested and we do not get the event on startup.
This has been resolved by setting the default of `focused` in the store to `true`.

Reply:
Thank you! There is an edge where iamb is started without focus, but I don't think it's possible to query the focus state from the terminal, and it's not a big deal.

Reply:
Np :)

@benjajaja, I was also thinking about that edge case this morning, but I wasn't able to think of an easy way to resolve it either...

The only feasible way I can think of is to execute `crossterm` on entry to main, and run login/sync in the background.

That would end up being a fairly significant change, but if you are interested I can try and tackle it in another PR?

Reply:
I don't understand how that would give us the focus-state of the terminal. But I'm not really worried about that edge case, this PR is much more important. Thanks again!